src/assets/js/protocol/protocol-details.js

```diff
@@ -0,0 +1,229 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+// create namespace
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+(function(doc) {
+    'use strict';
+
+    var Details = {};
+    var _count = 0;
+
+    /**
+     * isSupported
+     * - feature detection for HTML5 detail/summary support
+     * - // https://mathiasbynens.be/notes/html5-details-jquery#comment-35
+     */
+    Details.isSupported = (function() {
+        var el = doc.createElement('details');
+        var fake;
+        var root;
+        var diff;
+        if (!('open' in el)) {
+            return false;
+        }
+        root = doc.body || (function() {
+            var de = doc.documentElement;
+            fake = true;
+            return doc.insertBefore(doc.createElement('body'), de.firstElementChild || de.firstChild);
+        }());
+        el.innerHTML = '<summary>a</summary>b';
+        el.style.display = 'block';
+        root.appendChild(el);
+        diff = el.offsetHeight;
+        el.open = true;
+        diff = diff != el.offsetHeight;
+        root.removeChild(el);
+        if (fake) {
+            root.parentNode.removeChild(root);
+        }
+        return diff; //todo, how do I document return values
```

Comment:
How do I document functions with return values?

Reply:
You can to `@return`, like here https://github.com/mozilla/protocol/blob/master/src/assets/js/protocol/protocol-lang-switcher.js#L20